modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,108 @@ It also includes enhancements to the Eventing and Search services.
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
 
+[#release-605]
+== Release 6.0.5
+
+Couchbase Server 6.0.5, released in January 2021, is the fifth maintenance release in the 6.0.x series for Couchbase Server. 
+This release includes bug fixes related to many components.
+
+[#fixed-issues-605]
+=== Fixed Issues
+
+The notable fixes in this release are listed below.
+
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_v605-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-43416[MB-43416^]
+| *Summary*: In a high bucket density test (with 30 buckets) with replication on, goxdcr process on source cluster data node was observed to exit multiple times with `status 2` and a runtime error.
+
+| https://issues.couchbase.com/browse/MB-40846[MB-40846^]
+| *Summary*: Fixed an incorrect XDCR stream request rollback caused by a consumer ahead of producer error.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v605-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-41804[MB-41804^]
+| *Summary*: Fixed an issue where ephemeral buckets set to `auto_delete` with replicas configured could reach a deadlock due to eviction.
+
+| https://issues.couchbase.com/browse/MB-41255[MB-41255^]
+| *Summary*: Fixed an issue where the replication of deletes with XATTRS failed if a EWOULD_BLOCK occurred.
```

Comment:
"EWOULD_BLOCK" doesn't mean much to end-users. Suggest re-phrasing to "... with XATTRs failed if the affected document was non-resident."